validate_sample.py

```diff
@@ -93,7 +93,8 @@ def validate_arid(count, sample_row):
         _check_length(column, arid, count, maximum_length)
         if arid in ARID:
             print(f'Line {count}: {column}: {arid} is duplicated in sample file.')
-            status = 1
+            global error_count
```

Comment:
Why is error_count made global here instead of on line 37?

Reply:
It needs to be specified within each method that we're referring to the global variable, rather than a local copy.